app/src/main/java/org/oppia/android/app/devoptions/markchapterscompleted/MarkChaptersCompletedFragmentPresenter.kt

```diff
@@ -111,6 +122,11 @@ class MarkChaptersCompletedFragmentPresenter @Inject constructor(
     model: ChapterSummaryViewModel
   ) {
     binding.viewModel = model
+    if (getMarkChaptersCompletedViewModel().getItemList()
+      .count { it is ChapterSummaryViewModel && !it.checkIfChapterIsCompleted() } == 0
```

Comment:
This is a bit difficult to read.
Suggestion
```
    val notCompletedChapterCount = getMarkChaptersCompletedViewModel().getItemList().count {
      it is ChapterSummaryViewModel && !it.checkIfChapterIsCompleted()
    }
    if (notCompletedChapterCount == 0) {
      this.binding.isAllChecked = true
    }
```

Reply:
Done.